app/models/pet.rb

```diff
@@ -1,5 +1,11 @@
 class Pet < ApplicationRecord
-  validates_presence_of :image, :name, :approx_age, :sex, :status, :description, :shelter_id
+  validates_presence_of :image,
+                        :name,
+                        :approx_age,
+                        :sex,
+                        :status,
+                        :description,
+                        :shelter_id
```

Comment:
I think this is Ruby convention to list things vertically instead of horizontally when lists are longer than one or two. I prefer this way, but let me know if you have an objection.

Reply:
Yeah, I meant to change this but forgot.